@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -931,16 +948,19 @@ export class VisibleTileSet {
 
                         if (!checkedTiles.has(parentCode) && !renderedTiles.get(parentCode)) {
                             checkedTiles.add(parentCode);
-                            const parentTile = tileCache.get(parentCode, dataSource);
+
+                            const {
+                                offset,
+                                mortonCode
+                            } = TileOffsetUtils.extractOffsetAndMortonKeyFromKey(parentCode);
```

Comment:
So we still have this packing of offset into the morton code. Maybe we can remove this as well in a separate commit.

Reply:
I have checked it, this would much much bigger change, cause we utilize this system almost everywhere, that is why I was not so prone to introduce this change. We may still leave the old system in cache if you wish...